### ISSUE TITLE
Search features stuck

### ISSUE THREAD

**Issue by prescollar** [toopriddy/mytime#377]
It has been a month or so... but my wife and I have not been able to search anything except by “Name”. You cannot search for example by “Address”
Thank you.

**Comment by toopriddy** [comment]
Sorry for the trouble. Sadly this is an apple bug where they broke the search bar and navigation bar. This is a iOS bug which we are waiting on Apple to fix :(

You can however rotate the screen and tap on the buttons but when you rotate back they still do not work. Strange huh? Sadly Apple broke a lot in iOS 13

**Comment by prescollar** [comment]
Yeah... this update was big enough to have a lot of snags. And we have to find them all out for Apple. They should pay us!
Thanks for all you do. 
Been using this for years. (I haven’t contributed often enough! LOL) I saw one person comment one time on the idea of having some sort of planning setup for goals. Is there anything in the works possible... or is there something I haven’t learned already incorporated... 
Or the probable, It’s too complicated to even mess with. 
Tried setting up my own calendar, but it keeps wanting to integrate into my existing calendars, so I have to fight it off all the time. 
(Don’t mean to complicate your real life with stupid questions!)
Thanks again. 

pREsLeY :~}
(Charlton Presley Collar)

> On Oct 22, 2019, at 23:07, toopriddy <notifications@github.com> wrote:
> 

﻿Sorry for the trouble. Sadly this is an apple bug where they broke the search bar and navigation bar. This is a iOS bug which we are waiting on Apple to fix :(

You can however rotate the screen and tap on the buttons but when you rotate back they still do not work. Strange huh? Sadly Apple broke a lot in iOS 13

**Comment by toopriddy** [comment]
there was an update to MyTime 3.11.0 which fixes this and is the first in a long time because of lots of reasons. I hope that now that I have an AI buddy helping me that things will be a lot more stable
